services/ui-src/src/components/modals/AddEditReportModal.tsx

```diff
@@ -3,7 +3,8 @@ import { useContext, useState } from "react";
 import { Form, Modal, ReportContext } from "components";
 import { Spinner } from "@cmsgov/design-system";
 // form
-import formJson from "forms/addEditProgram/addEditProgram.json";
+import mcparFormJson from "forms/addEditProgram/addEditProgram.json";
+import mlrFormJson from "forms/addEditSubmission/addEditSubmission.json";
```

Comment:
Question: Should we name these `addEditMcparReport` and `addEditMlrReport` or something instead to keep it consistent? I see the name change for `AddeditReportModal` from `AddEditProgramModal`.

Reply:
This is actually related to my PR, I'll make the change on my end.